### PR TITLE
Add examples and lint

### PR DESCRIPTION
**Pull request by tannewt**

### CODE REVIEW

**Comment by makermelissa** [comment]
The FeatherWing example is not working (it isn't updating the display and no errors).

**Comment by makermelissa** [comment]
I realized there is no display-ruler.bmp in the examples and I used it off the IL0373 PR. Is this something we want to add to the examples?

**Comment by makermelissa** [comment]
The breakout isn't working either.

**Comment by makermelissa** [comment]
Ok, got the breakout working. I'll try and get the featherwing working now.

**Comment by tannewt** [comment]
I used all the same display-ruler.bmp file in my testing. I just forgot to copy it into each examples folder.

File: examples/ssd1675_2.13_monochrome.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import busio
+import displayio
+import adafruit_ssd1675
+
+displayio.release_displays()
+
+# This pinout works on a Metro and may need to be altered for other boards.
+
+# For breadboarding
+spi = busio.SPI(board.SCL, board.SDA)
```
Comment (makermelissa) [inline examples/ssd1675_2.13_monochrome.py]: Let's use the board.SPI() for the SPI pins instead of I2C.

**Review by makermelissa — CHANGES_REQUESTED** [review]
I could not get the FeatherWing example to work. The EPD library example works fine with just using only the DC and ECS pins like in this library, so I'm not sure why it isn't working.

File: examples/ssd1675_2.13_monochrome.py (hunk 1 above)
Comment (ladyada) [reply examples/ssd1675_2.13_monochrome.py]: yes please!

Comment (tannewt) [reply examples/ssd1675_2.13_monochrome.py]: Swapped to board.SPI, documented and tested using Feather M4 and breakout.

**Comment by tannewt** [comment]
Oops, I had additional local changes for the FeatherWing. It should work now. I also swapped to board.SPI. Thanks for the test and review!

**Review by makermelissa — APPROVED** [review]
Tested again. Works great. Thanks!

**Pull request merged**